README.md

```diff
@@ -784,177 +801,132 @@ resources. You can download it via `ilab model download`.
-1. foundational_skills/reasoning/unconventional_reasoning/lower_score_wins/qna.yaml: 5.67 -> 4.0 (-1.67)
-2. foundational_skills/reasoning/mathematical_reasoning/qna.yaml: 7.33 -> 6.0 (-1.33)
-3. foundational_skills/reasoning/temporal_reasoning/qna.yaml: 5.67 -> 4.67 (-1.0)
-
-### NO CHANGE (0.0 to 10.0):
-1. foundational_skills/reasoning/linguistics_reasoning/odd_one_out/qna.yaml (9.33)
-2. compositional_skills/grounded/linguistics/inclusion/qna.yaml (6.5)
-```
+- Below is an example of running MTBenchBranch with a local safetensors model directory:
+
+   ```bash
+   $ export ILAB_MODELS_DIR=$HOME/.local/share/instructlab/models
+   $ export ILAB_TAXONOMY_DIR=$HOME/.local/share/instructlab/taxonomy
+   $ ilab model evaluate --benchmark mt_bench_branch \
+      --model $ILAB_MODELS_DIR/instructlab/granite-7b-test \
+      --base-model $ILAB_MODELS_DIR/instructlab/granite-7b-lab \
+      --taxonomy-path $ILAB_TAXONOMY_DIR \
+      --branch rc \
+      --base-branch main
+   ...
+   # SKILL EVALUATION REPORT
+
+   ## BASE MODEL (SCORE)
+   /home/user/.cache/instructlab/models/instructlab/granite-7b-lab (5.78/10.0)
+
+   ## MODEL (SCORE)
+   /home/user/.cache/instructlab/models/instructlab/granite-7b-test (6.00/10.0)
+
+   ### IMPROVEMENTS (0.0 to 10.0):
+   1. foundational_skills/reasoning/linguistics_reasoning/object_identification/qna.yaml: 4.0 -> 6.67 (+2.67)
+   2. foundational_skills/reasoning/theory_of_mind/qna.yaml: 3.12 -> 4.0 (+0.88)
+   3. foundational_skills/reasoning/linguistics_reasoning/logical_sequence_of_words/qna.yaml: 9.33 -> 10.0 (+0.67)
+
+   ### REGRESSIONS (0.0 to 10.0):
+   1. foundational_skills/reasoning/unconventional_reasoning/lower_score_wins/qna.yaml: 5.67 -> 4.0 (-1.67)
+   2. foundational_skills/reasoning/mathematical_reasoning/qna.yaml: 7.33 -> 6.0 (-1.33)
+   3. foundational_skills/reasoning/temporal_reasoning/qna.yaml: 5.67 -> 4.67 (-1.0)
+
+   ### NO CHANGE (0.0 to 10.0):
+   1. foundational_skills/reasoning/linguistics_reasoning/odd_one_out/qna.yaml (9.33)
+   2. compositional_skills/grounded/linguistics/inclusion/qna.yaml (6.5)
+   ```
 
 ### 🍴 Serve the newly trained model
 
-1. Stop the server you have running by entering `ctrl+c` keys in the terminal running the server.
+If you have a server running, stop the server you have running by entering `ctrl+c` keys in the terminal running the server.
 
-   > **IMPORTANT**:
-   - 🍎 This step is only implemented for macOS with M-series chips (for now).
+- 🍎 This step is only implemented for macOS with M-series chips (for now).
 
-   - Before serving the newly trained model you must convert it to work with
-   the `ilab` cli. The `ilab model convert` command converts the new model into quantized [GGUF](https://medium.com/@sandyeep70/ggml-to-gguf-a-leap-in-language-model-file-formats-cd5d3a6058f9) format which is required by the server to host the model in the `ilab model serve` command.
+Before serving the newly trained model you must convert it to work with
```

Comment:
I would rephrase this somehow - AFAIK conversion is something only Mac _needs_ to do, not necessarily a "feature" that is Mac-only - cc @cdoern

Reply:
yes, mac only. I wouldnt call it a feature. the full pipeline does this for you